lib/types/composite_type.js

```diff
@@ -23,11 +23,9 @@ function wrapField(field, value) {
     return null;
   }
 
-  if (field.type === '*') {
-    if (field.hasOwnProperty('requires'))
-       return (value instanceof field.requires) ? value : new field.requires(value);
-    return value;
-  }
+  if (field.hasOwnProperty('requires'))
+    return (value instanceof field.requires) ? value : new field.requires(value);
```

Comment:
this is for cases like `terminus.Source/Target`, those are in fact composite types and were the initial use case for `requires`